cellmlmanip/model.py

```diff
@@ -387,6 +387,20 @@ def get_derived_quantities(self):
                               and node.get('variable_type', '') not in ('state', 'free', 'parameter')]
         return sorted(derived_quantities, key=lambda var: var.order_added)
 
+    def get_display_name(self, var, META='https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#'):
```

Comment:
Method arguments shouldn't be all-caps. That's for constants.

I'd called this `preferred_ontology` or similar.